src/lfx/src/lfx/components/files_and_knowledge/save_file.py

```diff
@@ -13,6 +13,20 @@
 from lfx.schema import Data, DataFrame, Message
 from lfx.services.deps import get_settings_service, get_storage_service, session_scope
 from lfx.template.field.base import Output
+from lfx.utils.validate_cloud import is_astra_cloud_environment
+
+
+def _get_storage_location_options():
+    """Get storage location options, filtering out Local if in Astra cloud environment."""
+    all_options = [
+        {"name": "Local", "icon": "hard-drive"},
+        {"name": "AWS", "icon": "Amazon"},
+        {"name": "Google Drive", "icon": "google"},
+    ]
+    # Filter out Local option if in Astra cloud environment
+    if is_astra_cloud_environment():
+        return [opt for opt in all_options if opt["name"] != "Local"]
+    return all_options
```

Comment:
more of a stylistic nit, feel free to ignore haha
```suggestion
all_options = [
    {"name": "AWS", "icon": "Amazon"},
    {"name": "Google Drive", "icon": "google"}
    ]
if is_astra_cloud_environment():
   return all_options
 return  [{"name": "Local", "icon": "hard-drive"}] + all_options
```